src/account-info/LogoutButton.js

```diff
@@ -27,7 +27,11 @@ class LogoutButton extends PureComponent {
   shutdownPUSH = async () => {
     const { auth, actions, pushToken } = this.props;
     if (pushToken !== '') {
-      await unregisterPush(auth, pushToken);
+      try {
+        await unregisterPush(auth, pushToken);
+      } catch (e) {
+        console.log('failed to unregister Push token', e); // eslint-disable-line
```

Comment:
Is this enough? Are you using it to silence the exception or?
When does that happen?